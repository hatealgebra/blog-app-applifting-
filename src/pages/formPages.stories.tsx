import React from "react";

import Login from "./login";

export const LoginFormPage = () => <Login />;

export default {
  title: "Pages/Form Pages",
  subcomponents: { Login },
};
