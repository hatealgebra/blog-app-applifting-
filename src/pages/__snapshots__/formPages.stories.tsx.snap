// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/Form Pages LoginFormPage smoke-test 1`] = `
<div class="sc-AOXSc ddQreF">
  <header variant="desktop"
          class="sc-cUfYYG eqofrP TopNavbar"
  >
    <div class="TopNavbar__container">
      <div class="TopNavbar__sub-container">
        <img src="static/media/src/images/cat-logo.svg"
             alt="cat logo"
             height="40px"
        >
        <nav class="sc-jOHGOj jHhANX">
          <a variant="classic"
             class="sc-jrkPcm eCMzXk"
             href="/"
          >
            Recent Articles
          </a>
          <a variant="classic"
             class="sc-jrkPcm eCMzXk"
             href="/about"
          >
            About
          </a>
        </nav>
      </div>
      <div class="TopNavbar__sub-container">
        <a class="sc-ipMvLY hcUcbw login-link"
           href="/login"
        >
          <span>
            Login
          </span>
          <svg stroke="currentColor"
               fill="none"
               stroke-width="0"
               viewbox="0 0 24 24"
               height="1em"
               width="1em"
               xmlns="http://www.w3.org/2000/svg"
               style="transform: rotate(180deg);"
          >
            <path stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  d="M10 19l-7-7m0 0l7-7m-7 7h18"
            >
            </path>
          </svg>
        </a>
      </div>
    </div>
  </header>
  <div class="sc-bCnriq feOXrx">
    <div class="sc-dPOtYP hhWvXd">
      <form class="sc-hhWzdI kDlOd">
        <h3>
          Log In
        </h3>
        <div class="sc-dmBjiZ ieOkoL">
          <label for="Email">
            Email
          </label>
          <input placeholder="me@example.com"
                 type="email"
                 id="Email"
                 class="sc-iBAaJG djhrVw"
                 value
          >
        </div>
        <div class="sc-dmBjiZ ieOkoL">
          <label for="Password">
            Password
          </label>
          <input placeholder="Enter the password"
                 type="password"
                 id="Password"
                 class="sc-iBAaJG djhrVw"
                 value
          >
        </div>
        <span class="error">
        </span>
        <button variant="standard"
                colortheme="primary"
                type="submit"
                class="sc-hLclGa dhTwVn"
        >
          Log In
        </button>
      </form>
    </div>
  </div>
</div>
`;
